delete.ps1

```diff
@@ -24,7 +24,7 @@ $WarningPreference = "Continue"
 # Correlation values
 $correlation = [PSCustomObject]@{
     CorrelationProperty = "personeelsNummer" # Has to match the name of the unique identifier
-    CorrelationValue    = $aRef.Id # Has to match the value of the unique identifier
+    CorrelationValue    = $aRef # Has to match the value of the unique identifier
 }
 
 # Change mapping here
```

Comment:
Please modify the mapping to no longer define $updateAccountFields and $storeAccountFields in the script. Instead, these should be defined in the field mapping using the Ps v2 connector.